lnbits/static/js/base.js

```diff
@@ -524,12 +523,7 @@ window.windowMixin = {
     } else {
       this.$q.dark.set(true)
     }
-    let reactions = this.$q.localStorage.getItem('lnbits.reactions')
-    if (reactions == null) {
-      this.$q.localStorage.set('lnbits.reactions', true)
-      reactions = true
-    }
-    this.reactionToggle = reactions
+    this.reactionChoice = this.$q.localStorage.getItem('lnbits.reactions')
```

Comment:
```suggestion
    this.reactionChoice = this.$q.localStorage.getItem('lnbits.reactions') || 'confettiBothSides'
```

- since the declaration is `reactionChoice: 'confettiBothSides'` I assume this should be the default value.